package.json

```diff
@@ -28,7 +28,7 @@
   ],
   "scripts": {
     "build": "run-s build:distfiles build:types",
-    "build:distfiles": "lerna run --parallel --prefix --concurrency=8 build",
+    "build:distfiles": "lerna run build",
```

Comment:
is parallel useless?

Reply:
yes, it's deprecated. `lerna` runs in parallel if it detects it can run in parallel.  there's some nuance there, but works for me